webpack.config.js

```diff
@@ -29,7 +29,7 @@ config.module = {
 
 // Overrides to disable CSS Modules for non-scss scripts, this means
 // CSS Modules will only be supported with .scss scripts
-// config.cssRuleConfig.use = ["style-loader", "css-loader"];
-// config.cssRuleConfig.test = /\.css$/;
+config.cssRuleConfig.use = ["style-loader", "css-loader"];
+config.cssRuleConfig.test = /\.css$/;
```

Comment:
@pirupius I'm not sure why the about config was commented out. Commenting the config above was breaking the custom Ethiopia date-picker's styling.

Reply:
Not sure either 🤔